README.md

```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
 * Python 2.6+ (for running with the launcher script)
-* Docker (for building and running tests)
+* Docker (for building and running tests, and for building documentation)
 
 ## Building Presto
 
-Presto is a standard Maven project. Simply run the following command from the project root directory:
+Presto is a standard Maven project. Simply run the following command from the
```

Comment:
These line wrapping changes are formatting changes, which should be done as a separate commit that only changes formatting (no content changes). It's hard to review content changes when formatting changes are mixed together.

Reply:
Thank you!